kite-data/kite-data-core/src/main/java/org/kitesdk/data/Formats.java

```diff
@@ -53,10 +53,18 @@ private Formats() {
       new CompressionType[] { Uncompressed });
 
   /**
-   * Return a {@link Format} for the format name specified. If
-   * {@code formatName} is not a valid name, an IllegalArgumentException is
-   * thrown. Currently the formats <q>avro</q>, <q>csv</q>, and <q>parquet</q>
-   * are supported. Format names are case sensitive.
+   * INPUTFORMAT: a mapreduce InputFormat (read-only).
```

Comment:
Yes. The writer implementation checks for supported formats and bails if it can't create one:

https://github.com/kite-sdk/kite/blob/master/kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/FileSystemWriter.java#L220

Good catch though, there should be a check in initialize and also one in the partitioned writer.